packages/cashier/src/Containers/deposit.jsx

```diff
@@ -52,11 +52,14 @@ const Deposit = ({
 }) => {
     React.useEffect(() => {
         setActiveTab(container);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    React.useEffect(() => {
         if (!is_virtual) {
             onMount();
         }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    }, [is_virtual, onMount]);
```

Comment:
We should call `onMount` only once. Remounting on account switch will happen in `cashier-store`. 
Adding `is_virtual` to dependency will call `onMount` twice during account switch, once from here and once from account switch listener in `cashier-store`

I think we don't need `is_virtual` condition here. `onMount` should be called irrespective of the account type(`real`/`virtual`).